File: OP_labs_cs/lab2_cs/lab2_cs/Abit.cs
using System;

namespace lab2_cs
{
    
    public class Abit
    {
        public string FullName { get; set; }
        public DateTime BirthDay { get; set; }
        public SexType Sex { get; set; }

        public Abit(string fullName, int day, int month, int year, SexType sex)
        {
            FullName = fullName;
            BirthDay = new DateTime(year, month, day);
            Sex = sex;
        }
        public int Age()
        {
            DateTime now = DateTime.Now;
            int age = now.Subtract(BirthDay).Days / 365;
            return age;
        }

        public string Info()
        {
            return string.Format("\nFull name: {0} \nBirthday: {1} \nSex: {2}", FullName, BirthDay.ToString("d"), Sex);
        }
        public enum SexType
        {
            Male,
            Female
        }
    }
}

File: OP_labs_py/lab2_py/Abit.py
from datetime import *


class Abit:
    def __init__(self, name: str, birth: str, sex: str):
        self.fullName = name
        self.birthDay = birth
        self.sexType = sex

    def get_info(self):
        return f"Fullname: {self.fullName}\nBirthday: {self.birthDay}\nSex: {self.sexType}\n"

    def get_age(self):
        birth = datetime.strptime(self.birthDay, "%d.%m.%Y")
        now = datetime.now()
        age = (now - birth).days/365
        return age
